draft-wang-ppm-dap-taskprov.md

```diff
@@ -568,7 +568,10 @@ by including a new logical role, called the Author. The Author is responsible
 for configuring Clients prior to task execution. For privacy we consider the
 Author to be under control of the adversary. It is therefore incumbent on
 protocol participants to verify the privacy parameters of a task before opting
-in.
+in. One risk introduced by the addition of the Author is the Author can
+configure tasks uniquely for a Client. If the Author colludes with the Leader,
+it can track the uploading activities of that Client, which leaks information
+about that Client, e.g., location.
```

Comment:
We can point to some mitigation, e.g., OHTTP, not sure if it's worth mentioning here.

Reply:
How does OHTTP mitigate this risk?

Reply:
Actually, by checking the task parameters, it will be mitigated slightly, e.g., by making sure min batch size is not too small, but the Author can still target the same group of clients for this attack.

Reply:
(moved to another thread)